index.js

```diff
@@ -92,7 +219,7 @@ export class Placement {
     }
 
     let classes = (element.className || "").split(" ");
-    if (classes.includes("loaded")) {
+    if (classes.indexOf("loaded") >= 0) {
```

Comment:
Just a note that `includes` requires a polyfill in IE11. That's why I removed it.